src/class_objectProcessor.py

```diff
@@ -124,21 +134,25 @@ def checkackdata(self, data):
             logger.info('This object is an acknowledgement bound for me.')
             del shared.ackdataForWhichImWatching[data[readPosition:]]
             sqlExecute(
-                'UPDATE sent SET status=?, lastactiontime=?'
-                ' WHERE ackdata=?',
+                'UPDATE sent SET status=?, lastactiontime=? WHERE ackdata=?',
                 'ackreceived', int(time.time()), data[readPosition:])
-            queues.UISignalQueue.put((
-                'updateSentItemStatusByAckdata',
-                (data[readPosition:],
-                 tr._translate(
-                     "MainWindow",
-                     "Acknowledgement of the message received %1"
-                 ).arg(l10n.formatTimestamp()))
-            ))
+
+            queues.UISignalQueue.put(
+                (
+                    'updateSentItemStatusByAckdata',
+                    (
+                        data[readPosition:],
+                        tr._translate(
+                            "MainWindow",
+                            "Acknowledgement of the message received %1").arg(l10n.formatTimestamp())
+                    )
+                )
+            )
```

Comment:
What was wrong here? Aesthetics?

Reply:
Yes. I've explained my feelings before about hanging indentation and how I've added it as a strawman to the 'bikeshedding' section of the docs, namely short, simple functions can close all braces on one line while data structures and larger, more complex nested functions are aligned to help the reader.